src/quanteda/generate_return_series.py

```diff
@@ -0,0 +1,53 @@
+import pandas as pd
+import numpy as np
+
+def generate_return_series(
+    expected_annual_return, 
+    annual_volatility, 
+    length=365, 
```

Comment:
adopting Doris's recommendation:
```suggestion
    n_rows=365, 
```